src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  let basicDiscount = 50;
```

Comment:
i'd use two variables and avoid rewriting this one
it is not a basic discount if it changes after all